packages/react-components/react-virtualizer/src/utilities/createResizeObserverFromDocument.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * A utility method that creates a ResizeObserver from a target document
+ * @param targetDocument - document to use to create the ResizeObserver
+ * @param callback  - https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver/ResizeObserver#callback
+ * @returns a ResizeObserver instance or null if the global does not exist on the document
+ */
+export function createResizeObserverFromDocument(
```

Comment:
I duplicated this - it's not possible to re-export from any package since ResizeObserver types don't exist on our TS Minbar

Reply:
Would be nice to see a comment with a FIXME tag saying that this would not be necessary in a future version of TS